search/lib/api/wfs.js

```diff
@@ -26,6 +27,8 @@ const env = {};
 env.BROWSE_PATH = process.env.BROWSE_PATH;
 env.CONFORMANCE_RESPONSE = CONFORMANCE_RESPONSE;
 
+Object.fromEntries = l => l.reduce((a, [k, v]) => ({ ...a, [k]: v }), {});
```

Comment:
Can this be it's own function instead of monkeypatching the `Object` prototype?

Reply:
Granted, this is a bit of a hack job, but it's only temporary.  `fromEntries` is a core function in Node 12+, and this is just a hack for Node 10. Given that we have to move off 10 in the near future (EOL 2 months from now) I'm inclined to wait and delete this function when we upgrade.